games.yaml

```diff
@@ -2786,6 +2855,14 @@
           image: http://www.oolite.org/images/gallery/coriolis!jpg
         - url: http://www.oolite.org/images/gallery/full/advanced-nav-array
           image: http://www.oolite.org/images/gallery/advanced-nav-array
+  clones:
+    - name: Mission ELITE: The Battle For Birera
```

Comment:
You have to put quote marks around string if it contains a colon, so should be `- name: "Mission ELITE: The Battle For Birera"`